src/containers/templates/CreateTask/CreateTask.js

```diff
@@ -57,20 +116,34 @@ class CreateTask extends Component {
             )}
           </div>*/}
 
+        {this.state.message &&
+          <div>
+            {this.state.message}
+          </div>}
+
         <div className="text-center margin-top">Set Reminder Frequency</div>
 
         <div className="dropDown margin-top">
-          <Input className="margin" type="select" defaultValue="1">
-            <option className="text-center" value="1">
-              Daily
+          <Input
+            className="margin"
+            type="select"
+            value={this.state.stagedFrequency}
+            onChange={this.onFrequencyChange.bind(this)}
+          >
+            <option className="text-center" value="none">
+              None
             </option>
-            <option value="2">Weekly</option>
-            <option value="3">Monthly</option>
+            <option value="daily">Daily</option>
+            <option value="weekly">Weekly</option>
+            <option value="monthly">Monthly</option>
           </Input>
         </div>
 
         <div className="exp-button width-80 btn-blue">
-          <BaseButton title={'SET REMINDER'} />
+          <BaseButton
+            title={'SET REMINDER'}
+            onClick={this.setReminder.bind(this)}
```

Comment:
you can just declare `setReminder` as class properties so you don't need to bind all the time,

```
setReminder = () => {}
```